### PR TITLE
add owners file

### PR DESCRIPTION
**Pull request by rkpagadala**
**What this PR does / why we need it**:
Required owner file for approval and auto merging

**Which issue this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*: fixes #

**Special notes for your reviewer**:

**Release note**:

```release-note
```

### CODE REVIEW

File: OWNERS [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+  - sebastienvas
+  - geeknoid
+  - rshriram
+approvers:
+  - qiwzhang
+  - lizan
+  - sebastienvas
+  - geeknoid
+  - rshriram
+  - JimmyCYJ
```
Comment (lizan) [inline OWNERS]: should Jimmy also listed in reviewers?

Comment (rkpagadala) [reply OWNERS]: fixed

**Comment by rkpagadala** [comment]
@lizan this PR needs manual merging. Going forward you can add /approve and /lgtm for auto merge.

**Pull request merged**